polygeist/lib/Dialect/Polygeist/Transforms/LoopInternalization.cpp

```diff
@@ -1300,10 +1308,11 @@ void LoopInternalization::transform(FunctionOpInterface func,
   sycl::populateLocalID(localIDs, numDims, builder, func.getLoc());
 
   // Reserve static shared local memory for this function.
+  auto gpuModule = func->getParentOfType<gpu::GPUModuleOp>();
```

Comment:
Nit: Should we assert for `gpuModule` here or in `getWorkGroupSharedLocalMemory`?

Reply:
I think here's better

Reply:
done.